client/gui/config.ts

```diff
@@ -11,16 +28,15 @@ export const _gui = {
   is_table_visible: false,
   is_label_bar_visible: true,
   column_visibilities: new Array(10).fill(true),
-  textarea_height: 238,
+  textarea_height: "238px",
```

Comment:
We're just using this to do
```ts
$(...).css("height", config.textarea_height);
```
where `238` and `'238px"` have the same meaning.  I changed this, though, because we later store
```ts
config.textarea_height = $(...).css("height");
```
which always gives back a string.